src/app.py

```diff
@@ -1,26 +1,51 @@
 import json
+import logging
 import os
+import schedule
+import time
 from article import Article
 from publication import Publication
 from pymongo import MongoClient, UpdateOne
 
+# Open publications json file
 with open("publications.json") as f:
     publications_json = json.load(f)["publications"]
 
-publications = []
-articles = []
-for publication in publications_json:
-    pub = Publication(publication)
-    for entry in pub.get_feed():
-        articles.append(Article(entry, publication['slug']).serialize())
-    publications.append(pub.serialize())
+MONGO_ADDRESS = os.getenv("MONGO_ADDRESS")
 
-publication_upserts = [ UpdateOne({'_id':p['_id']}, {'$setOnInsert':p}, upsert=True) for p in publications ]
-article_upserts = [ UpdateOne({'articleUrl':a['articleUrl']}, {'$setOnInsert':a}, upsert=True) for a in articles ]
+# Get serialized publications
+publications = [Publication(p).serialize() for p in publications_json]
+publication_upserts = [
+    UpdateOne({"_id": p["_id"]}, {"$setOnInsert": p}, upsert=True) for p in publications
+]
```

Comment:
i take it back... Orko code is excellent documentation

Reply:
list comprehensions are a bit excessive here lol